engine/execution/computation/manager.go

```diff
@@ -313,7 +313,7 @@ func (e *Manager) ComputeBlock(
 
 	e.edCache.Insert(block.Block.Header, blobTree)
 	e.log.Info().Hex("block_id", logging.Entity(block.Block)).Hex("execution_data_id", rootID[:]).Msg("execution data ID computed")
-	// result.ExecutionDataID = rootID
+	result.ExecutionDataID = rootID
```

Comment:
I don't think we want to merge this into master with this yet, but it's needed for the integration tests. Maybe we should merge this into a execution data sync feature branch?